crawl-ref/source/potion.cc

```diff
@@ -73,7 +69,11 @@ class PotionCuring : public PotionEffect
             return true;
         }
 
-        return false;
```

Comment:
Fair call, I should have mentioned this in the issue body.

On 22 April 2015 at 16:43, Alan Malloy <notifications@github.com> wrote:

> In crawl-ref/source/potion.cc
> <https://github.com/crawl/crawl/pull/11#discussion_r28848105>:
>
> > @@ -73,7 +69,11 @@ class PotionCuring : public PotionEffect
> >              return true;
> >          }
> >
> > -        return false;
>
> I think this makes it impossible for players to quaff a heal_wounds
> manually when they're at full HP and not rotted. Maybe that's a good
> change, but I'm not sure it's a good idea to sneak that in with a gozag
> change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crawl/crawl/pull/11/files#r28848105>.
>